server/service/src/processors/transfer/README.md

```diff
@@ -0,0 +1,62 @@
+# Transfers
+
+Pair records that belong to two different stores but are intrinsically linked together are called `transfers`:
+
+`RequestRequisition` -> `ResponseRequisition`
+
+`OutboundShipment` -> `InboundShipment`
+
+The term `transfer` arises from the need to `transfer` these records from one `sync site` to another, in a scenario where a pair of linked transfer records belong to two different `sites`.
```

Comment:
the initial definition doesn't make the distinction about being on a different site. If that's key to being a 'transfer' then it should be included too